### DISCUSSION TITLE
Having repositry management issues, can't pull or push: fatal could not read from remote repository

### DISCUSSION THREAD

**Discussion by lewisalvigno** [community/community#73830] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Having repositry management issues, can't pull or push: fatal could not read from remote repository

**Comment by b8zeek** [comment, marked as answer]
This usually occurs because of the authentication issues. Not sure which authentication method you're using, in case of HTTPS update your credentials, in case of SSH do set up SSH keys correctly.

Good luck. 🍺 

🍀
